modules/test/playwright/helpers/ApiHelpers.ts

```diff
@@ -42,6 +45,22 @@ export class ApiHelpers {
 		return response.json();
 	}
 
+	async getResponse(url: string) {
+		const response = await this.page.request.get(url, {
+			headers: await this.getHeader(),
+		});
+
+		return response;
+	}
+
+	async putResponse(url: string) {
+		const response = await this.page.request.put(url, {
+			headers: await this.getHeader(),
+		});
+
+		return response;
+	}
+
 	async post(url: string, data: DataObject) {
```

Comment:
Is this method needed or could `postResponse` be used?

Reply:
in some methods we get data directly from json response, other response elements are not needed, I'll reuse the code of postResponse to call it